### DISCUSSION TITLE
non-integer initial_size in add_population()

### DISCUSSION THREAD

**Discussion by mshpak76** [tskit-dev/msprime#2253] [General]
It occurred to me that due to a script that multiplies population sizes by a scaling factor, I'm sometimes setting initial_size to non-integer values when specifying input arguments for demography.add_population(), e.g. I may have 1001.52 rather than 1001 or 1002. 

I'm wondering if this may result in any issues when running sim_ancestry. No error messages were thrown and my results look plausible, but I don't know whether the code sim_ancestry etc rounds non-integer values, or if this may be a source of incorrect results. Unfortunately, I only discovered that I had non-integer arguments in several of the simulations that I ran after the fact, and I don't know if this may be a problem.

**Comment by jeromekelleher** [comment]
Floating point values are totally fine - everything is a continuous approximation of discrete population sizes under the hood anyway, so no need to worry about rounding etc
